app/services/storage.js

```diff
@@ -373,6 +374,77 @@ export class ProfileStorage {
     return out;
   }
 
+  /**
+   * Search for places that match title, URL, or saved content
+   * since the specified timestamp.
+   *
+   * Snippets in the returned results are guaranteed to be HTML-escaped.
```

Comment:
What does this mean?  That they can be interpolated into HTML as values?  Would other consumers find this irritating?